scripts/add-list-items.js

```diff
@@ -0,0 +1,17 @@
+export default function addListItem(event, input, list) {
+  // Prevent button from scrolling up
+  event.preventDefault();
+
+  // Remove empty space from input
+  const inputValue = input.value.trim();
+
+  // Make sure input is not empty
+  if (inputValue !== '') {
+    const listItem = document.createElement('li');
+    listItem.textContent = inputValue;
+    list.appendChild(listItem);
+
+    // Empty input after
+    input.value = '';
```

Comment:
if your function param becomes a simple string, `inputValue`, don't forget to take care of this input-reset outside of this function!